Modules/Messages/src/Messages.Domain/Ids/FileId.cs

```diff
@@ -22,9 +22,9 @@ public static FileId Parse(string stringValue)
         return new FileId(stringValue);
     }
 
-    public static bool IsValid(string stringValue)
+    public static bool IsValid(string? stringValue)
```

Comment:
First you make the value nullable, and then below you suppress the nullable warning - it's contradicting I think.